test/openapi3.test.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const assert = require('assert');
+const openapi3 = require('../lib/openapi3');
+const options =
+    {
+        'omitBody': true,
+        'expandBody': false
+    };
+const operation =
+    {
+        'requestBody': {
+            'required': false,
+            'description': 'This is a description'
+        }
+    };
+const bodyParameter =
+    {
+        'refName': 'name of reference',
+        'schema': {
+            'type': 'object',
+            'properties':
+                {
+                    'id':
+                        {
+                            'type': 'string',
+                            'description': 'The fake ID'
+                        }
+                }
+        }
+    };
+
+const noOptions = {
+    'options': {},
+    'operation': operation,
+    'parameters': [],
+    'bodyParameter': bodyParameter,
+    'translations': {
+        'indent': ''
+    }
+};
+
+const noOperation = {
+    'options': options,
+    'operation': {},
+    'parameters': [],
+    'bodyParameter': bodyParameter,
+    'translations': {
+        'indent': ''
+    }
+};
+
+const goodData =
+    {
+        'options': options,
+        'operation': operation,
+        'parameters': [],
+        'bodyParameter': bodyParameter,
+        'translations': {
+            'indent': ''
+        }
+    };
+
+describe('openapi3 tests', () => {
+    describe('fakeBodyParameter', () => {
+        it('should handle empty options', () => {
+            openapi3.fakeBodyParameter(noOptions);
```

Comment:
What are we testing here and on L.71, that the function doesn't `throw`?

Reply:
I see what you mean. What happened is I kept getting the error, "cannot read property 'required' of undefined' while trying to write the test on line 74, so I thought It could potentially be helpful.

That said, my main goal here was to get my feet wet because I'd like to help write some tests for this PR https://github.com/Mermade/widdershins/pull/154 since I'm having related issues. So I don't mind removing these if you feel they're not helpful.

Reply:
No, I was just trying to ask if that test as written failed without your guard-condition `if` introduced in the `openapi3.js` file.

Reply:
Tests for #154 would be fantastic. Getting that PR sorted one way or another is basically holding up the next release.

Reply:
Oh I see. The test one line 71 does fail without that `if`. This one works, but I just threw it in. :)

Great! That's on my to-do list.

Reply:
Could we use [assert.doesNotThrow](https://nodejs.org/api/assert.html#assert_assert_doesnotthrow_fn_error_message) here just to make clear what we're testing?